jwst/outlier_detection/outlier_detection.py

```diff
@@ -190,6 +202,12 @@ def do_detection(self):
         # clean-up (just to be explicit about being finished with
         # these results)
         del median_model, blot_models
+        self.output_list.extend(blot_files)
+        if not pars['save_intermediate_results']:
+            for fle in self.output_list:
```

Comment:
Is there any reason to not use the object name `file` here, instead of the somewhat cryptic `fle`? Or is `file` something special within python?

Reply:
it is an old costume of mine to avoid using file (but I don't remember where it came from) I can change it and it should not be an issue for Python.